android-components/components/feature/app-links/src/test/java/mozilla/components/feature/app/links/AppLinksInterceptorTest.kt

```diff
@@ -644,4 +869,38 @@ class AppLinksInterceptorTest {
         assertFalse(inUserDoNotIntercept("https://example.com", testIntent))
         assertFalse(inUserDoNotIntercept("https://test.com", testIntent))
     }
+
+    @Test
+    fun `WHEN request is redirecting to external app quickly THEN request is not intercepted`() {
+        appLinksInterceptor = AppLinksInterceptor(
+            context = mockContext,
+            interceptLinkClicks = true,
+            launchInApp = { true },
+            useCases = mockUseCases,
+        )
+
+        var response = appLinksInterceptor.onLoadRequest(
+            mockEngineSession,
+            webUrlWithAppLink,
+            null,
+            true,
+            false,
+            false,
+            false,
+            false,
+        )
+        assert(response is RequestInterceptor.InterceptionResponse.AppIntent)
```

Comment:
I see this is true throughout this file, but I have had problems with `assert` not actually failing tests since it isn't part of the JUnit framework. `assertTrue` would be a better bet

Reply:
Sure, I'll update to asserTrue for the newly added test.  Thanks